### ISSUE TITLE
Icon request: icon-sourceforge

### ISSUE THREAD

**Issue by lifeparticle** [FortAwesome/Font-Awesome#1178]
Brand Icons

**Comment by borisbaldassari** [comment]
+1

**Comment by christianhujer** [comment]
+1

**Comment by josepsanzcamp** [comment]
+1

**Comment by mwwhited** [comment]
+1

**Comment by halitalf** [comment]
+1

**Comment by Maffblaster** [comment]
+1

**Comment by omarz1** [comment]
Hello. Any documentation to how this can be done?

**Comment by ili-codes** [comment]
How come this isn't a thing yet?

**Comment by ili-codes** [comment]
+1

**Comment by thumbapps** [comment]
Yes, it's one of the most popular free hosting for a lot of open source and/or free software.

**Comment by noraj** [comment]
Wow I'm surprised there is Github, Gitlab, Bitbucket and not sourceforge, this issue is open since 2013. Sorry for the necrobumping notification but it looks like people in 2018 are still looking for a sourceforge icon.

**Comment by leoz** [comment]
+1

**Comment by noraj** [comment]
@leoz and in general everyone, there is a "Thumbs up" button instead of writting a "+1" comment that will notify 15 people.

![image](https://user-images.githubusercontent.com/16578570/51128961-6a709680-1829-11e9-8dc9-555accff9be3.png)

**Comment by leoz** [comment]
Gotcha, @noraj

**Comment by Fsu0413** [comment]
+1

**Comment by Neustradamus** [comment]
Sourceforge exists since 1999 and it is very used, it is like GitHub but what icon? SF or other?
- https://www.google.com/search?q=sourceforge+icon

cc: @tagliala.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
